src/components/ActNowButtonCard/BaseCard.js

```diff
@@ -0,0 +1,44 @@
+import React from "react";
+import PropTypes from "prop-types";
+
+import { CardContent, Typography, Grid } from "@material-ui/core";
+import CloseIcon from "@material-ui/icons/Close";
+
+import useStyles from "./useStyles";
+
+const BaseCard = ({ closeCard, title, description, children }) => {
+  const classes = useStyles();
+
+  return (
+    <CardContent className={classes.cardContent}>
+      <CloseIcon onClick={closeCard} className={classes.closeIcon} />
```

Comment:
It's good idea to keep event names as `onSomething`... so this should be `onClose` or `onCloseCard` if you want to be specific.